docs/scanner-quickstart.md

```diff
@@ -409,31 +402,41 @@ autoUpdate:
+- copying the token over to Forta App as described in the [scanner pool management](scanner-pools.md) page of the docs.
 
-To register your node to the registry contract, you can run `forta register --owner-address <address>`. **Make sure you have set the `chainId` in your config.yml correctly before executing this.** Your scan node can be registered only once and to scan a specific chain. The owner wallet address needs to be a different address than your scan node address.
+Alternatively, you can
 
-!!! warning "Action requires funds"
-    You need to fund your scan node address with some Polygon (Mainnet) MATIC to be able to send this transaction. You can find out your scan node address with `forta account address`.
+- do `forta authorize pool --id <your-pool-id> --polygonscan`,
+- find out the `ScannerPoolRegistry` contract address from the [smart contract addresses page](smart-contracts.md),
+- visit it on Polygonscan,
+- visit "Contract" tab and find "Write as Proxy",
+- click on "Connect to Web3" to connect your wallet,
+- find the `registerScannerNode` method, copy the values from the first step, send the transaction.
 
 ## Stake FORT
 
-To ensure network reliablity, Forta Network requires staking FORT tokens on your node. You can follow the [staking guide](https://docs.forta.network/en/latest/stake-on-scan-node/) to learn how to manage stake for a node.
+To ensure network reliablity, Forta Network requires staking FORT tokens on your node. You can follow the guide in the [Mange your scanner pools](https://docs.forta.network/en/latest/scanner-pools/) page to find out how to manage stake for nodes.
 
-!!! warning "All nodes require stake"
-    Unstaked nodes node will not be assigned any detection bots and will not generate any rewards.
+!!! warning "All pools require stake"
+    Scan nodes in the unstaked or understaked pools will not be assigned any detection bots and will not generate any rewards.
 
 Forta Network makes good use of the stake by enforcing two main mechanisms:
 
-- **Rewarding:** Node operators are incentivized with rewards to ensure that their Forta nodes are running with good health and as expected. To gain rewards, a node operator must `deposit()` the minimum amount of FORT required using the staking contract.
-- **Slashing:** Node operators are discouraged from harmful actions. Upon detection, they lose rewards and a determined amount is removed from the deposited stake. This can cause the staked amount to go under minimum required and the node to enter into disabled state.
+- **Rewarding:** Node operators are incentivized with rewards to ensure that their Forta nodes are running with good health and as expected.
+- **Slashing:** Node operators are discouraged from harmful actions. Upon detection, they lose rewards and a specific portion of the is removed from the deposited stake. This can cause the staked amount to go under minimum required and the node to enter into disabled state.
```

Comment:
specific portion of the stake?

Reply:
would the node enter into disabled state, or could the pool also become disabled?

Reply:
Nice catch! Clarifying that.